docs/conf.py

```diff
@@ -16,8 +16,8 @@
 
 # basic project information
 project = 'ddSMT'
-copyright = '2013-2021, the authors of ddSMT'
-author = 'The authors of ddSMT'
+copyright = '2013-2021, ddSMT authors'
+author = 'ddSMT authors'
```

Comment:
Is there a specific reason for this change? Or simply personal preference?

Reply:
Just preference, I think it sounds better but I don't know why. Should I drop it?

Reply:
We want to keep as is, so yes, please drop it.